src/v/config/configuration.cc

```diff
@@ -496,12 +496,13 @@ configuration::configuration()
   , target_quota_byte_rate(
       *this,
       "target_quota_byte_rate",
-      "Target request size quota byte rate (bytes per second) - 2GB default",
+      "Target request size quota byte rate (bytes per second) - disabled "
```

Comment:
What does "disabled" mean in this description? It's disabled by default? Or `0` = `disabled`?

Reply:
I would simply remove the default value from the description. In docs website we can add this extra info. 

If you really want to have it in the description maybe something like 
"Target request size quota byte rate (bytes per second) - (default: "0" - disabled)

Reply:
Thank you! Sounds good, I've removed the ending now.